### PR TITLE
header_options moved to main_run script

### LINKED ISSUE #56: Generated scripts do not work when shieldhit not in PATH
When working on prometheus cluster users usually add certain numbers of programs to PATH variable by running: `source $PLG_GROUPS_SHARED/plggccbmc/software/setup.sh`.
Let us assume we want to run Fluka simulation (which uses `rfluka` tool).
In this situation when running first  `source $PLG_GROUPS_SHARED/plggccbmc/software/setup.sh` and second generated `submit.sh`, the information where to find `rfluka` is passed via `PATH` to all executors.

Imagine a user which doesn't have nice machinery like: `source $PLG_GROUPS_SHARED/plggccbmc/software/setup.sh` but has SHIELDHIT installed in his private directory.

User would like to provide an option specifying a location of SH12A binary (i.e. via `export PATH=$PATH:.....`. At first glance `--scheduler_options` seems to be suitable for that.

It seems that it can be done by passing a file to `--scheduler_options`, but it seems not to work with Torque:

```./generatemc.pyz -p 14000 -j 5 -w w/ -s ./1.txt ./sh/sh/shieldhit/examples/simple/```

with settings:
```
[plgkongruencj@zeus plgkongruencj]$ cat 1.txt 
#PBS -A qcgdevtests3 
#PBS -l walltime=00:30:33
export PATH=$PATH:/mnt/lustre/scratch2/people/plgkongruencj/sh/sh/shieldhit/simulation/shield_hit/
```

Will generate following files:
```
[plgkongruencj@zeus plgkongruencj]$ cat w/run_20170102_222658/submit.sh 
#!/usr/bin/env bash
#PBS -A qcgdevtests3 
#PBS -l walltime=00:30:33
export PATH=$PATH:/mnt/lustre/scratch2/people/plgkongruencj/sh/sh/shieldhit/simulation/shield_hit/

qsub  -t 1-5 /mnt/lustre/scratch2/people/plgkongruencj/w/run_20170102_222658/workspace/main_run.sh
[plgkongruencj@zeus plgkongruencj]$ cat w/run_20170102_222658/workspace/main_run.sh 
#!/usr/bin/env bash

# Exit immediately if a simple command exits with a non-zero status.
set -e

# Run individual jobs
/mnt/lustre/scratch2/people/plgkongruencj/w/run_20170102_222658/workspace/job_`printf %04d $PBS_ARRAYID`/run.sh
```

The problem is that contents of `1.txt` (PATH modification, PBS options) went to script which calls `sbatch`. According to documentation https://docs.cyfronet.pl/pages/viewpage.action?pageId=4260614 it should go to the file which is called by sbatch, mainly main_run.sh

With following manual modification everything works well:
```[plgkongruencj@zeus plgkongruencj]$ cat w/run_20170102_212940/submit.sh 
#!/usr/bin/env bash

qsub  -t 1-5 /mnt/lustre/scratch2/people/plgkongruencj/w/run_20170102_212940/workspace/main_run.sh
[plgkongruencj@zeus plgkongruencj]$ cat w/run_20170102_212940/workspace/main_run.sh 
#!/usr/bin/env bash

#PBS -A qcgdevtests3 
#PBS -l walltime=00:30:33
export PATH=$PATH:/mnt/lustre/scratch2/people/plgkongruencj/sh/sh/shieldhit/simulation/shield_hit/

# Exit immediately if a simple command exits with a non-zero status.
set -e

# Run individual jobs
/mnt/lustre/scratch2/people/plgkongruencj/w/run_20170102_212940/workspace/job_`printf %04d $PBS_ARRAYID`/run.sh
```

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #56

### CODE REVIEW

**Comment by grzanka** [comment]
@ant6 can you check wercker config ?

**Comment by ant6** [comment]
@grzanka i'll try

**Comment by ant6** [comment]
It's strange - locally your solution with 'versioneer install' works:
```py
(tox27){17-01-02 23:25} ~/PycharmProjects/mcpartools@feature/56-path-stuff✗✗✗✗✗✗ ant6% ➜ pip install -r tests/requirements-test.txt

// installation...

(tox27){17-01-02 23:25} ~/PycharmProjects/mcpartools@feature/56-path-stuff✗✗✗✗✗✗ ant6% ➜ tox -e py27
GLOB sdist-make: /home/ant6/PycharmProjects/mcpartools/setup.py
ERROR: invocation failed (exit code 1), logfile: /home/ant6/PycharmProjects/mcpartools/.tox/log/tox-0.log
ERROR: actionid: tox
msg: packaging
cmdargs: ['/home/ant6/.virtualenvs/tox27/bin/python', local('/home/ant6/PycharmProjects/mcpartools/setup.py'), 'sdist', '--formats=zip', '--dist-dir', local('/home/ant6/PycharmProjects/mcpartools/.tox/dist')]
env: None

Traceback (most recent call last):
  File "setup.py", line 116, in <module>
    version=get_version(),
  File "setup.py", line 89, in get_version
    setup_versioneer()
  File "setup.py", line 33, in setup_versioneer
    versioneer.get_version()
AttributeError: 'module' object has no attribute 'get_version'

ERROR: FAIL could not package project - v = InvocationError('/home/ant6/.virtualenvs/tox27/bin/python /home/ant6/PycharmProjects/mcpartools/setup.py sdist --formats=zip --dist-dir /home/ant6/PycharmProjects/mcpartools/.tox/dist (see /home/ant6/PycharmProjects/mcpartools/.tox/log/tox-0.log)', 1)
(tox27){17-01-02 23:25} ~/PycharmProjects/mcpartools@feature/56-path-stuff✗✗✗✗✗✗ ant6% ➜ versioneer
Usage: versioneer install
(tox27){17-01-02 23:25} ~/PycharmProjects/mcpartools@feature/56-path-stuff✗✗✗✗✗✗ ant6% ➜ versioneer install
versioneer.py (0.18) installed into local tree
Now running 'versioneer.py setup' to install the generated files..
 creating mcpartools/_version.py
 mcpartools/__init__.py unmodified
 appending 'versioneer.py' to MANIFEST.in
 versionfile_source already in MANIFEST.in
(tox27){17-01-02 23:25} ~/PycharmProjects/mcpartools@feature/56-path-stuff✗✗✗✗✗✗ ant6% ➜ tox -e py27 

// tests pass  
```

**Comment by grzanka** [comment]
It looks that it is due to new version of versioneer https://github.com/warner/python-versioneer/releases/tag/0.18

**Comment by ant6** [comment]
btw- can't we just just `pytest` instead of making tox build env etc.?

**Comment by grzanka** [comment]
Well, yes, we could consider making pytest, but how would you easily handle multiple python versions on travis - its simple with tox.

**Comment by grzanka** [comment]
It looks that `versioneer.get_version()` is gone in 0.18 version, while in 0.17 it was present.

Check https://github.com/DataMedSci/mcpartools/blob/master/setup.py#L33

There should be some way to fix setup.py

**Comment by ant6** [comment]
Well, `build` on Wercker would be faster - just about this.

BTW - milestone 0.18 has one isssue... https://github.com/warner/python-versioneer/milestone/5?closed=1
and it was a fix for windows :|

**Comment by ant6** [comment]
@grzanka it can't be just gone(?) My local versioneer.py has "0.18" on the top of it and contains all `get_version` functions :|

**Comment by grzanka** [comment]
In fact there are more changes in versioneer:
https://github.com/warner/python-versioneer/compare/0.17...master

**Comment by grzanka** [comment]
I suspect another problem. Let us assume that you haven't typed yet `versioneer install`.
With versioneer<0.18 if you type: `import versioneer` then you have an error. With versioneer 0.18 you get good import.

I suspect that:
 - versioneer < 0.18 is just a binary (executable versioneer file) and NOT a python module
 - versioneer == 0.18 is just a binary (executable versioneer file) and a python module

This is causing a confusion (version == 0.18) with import versioneer as we can import local file versioneer.py (which has get_version() method) or system-wide versioneer package (which doesn't have get_version() method).

**Comment by grzanka** [comment]
@ant6 I've fixed versioneer version to <0.18, but I don't like this solution.
Can you check if for other projects: beprof, pymchelper, pytrip the same issue exists ?

File: wercker.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ build:
           echo "python version $(python --version) running"
           echo "pip version $(pip --version) running"
 
+    # generate versioneer files
```
Comment (grzanka) [inline wercker.yml]: No, it can be removed. Can you do it ? Also `.travis/install_linux.sh` can be reverted

**Pull request merged**